packages/graphql/src/schema/get-where-fields.ts

```diff
@@ -126,30 +124,25 @@ function getWhereFields({
             }
 
             if (["String", "ID"].includes(f.typeMeta.name)) {
-                const matchesSetting: boolean | undefined = features?.filters?.[f.typeMeta.name]?.MATCHES;
-                if (matchesSetting !== undefined) {
-                    if (matchesSetting === true) {
-                        res[`${f.fieldName}_MATCHES`] = { type: "String", directives: deprecatedDirectives };
-                    }
-                } else if (enableRegex) {
-                    // TODO: This is deprecated. To be removed in 4.0.0.
-                    res[`${f.fieldName}_MATCHES`] = { type: "String", directives: deprecatedDirectives };
-                }
-
-                const stringWhereOperators = ["_CONTAINS", "_STARTS_WITH", "_ENDS_WITH"];
+                const stringWhereOperators: { comparator: string; typeName: string }[] = [
```

Comment:
**nitpick, not important**: For these complex object I find array types to be a bit more readable as
 ```ts
Array<{ comparator: string; typeName: string }>
```